### PR TITLE
[KYUUBI #312] Support run kyuubi in foreground

### LINKED ISSUE #312: [Feature Request] Support run kyuubi-server in the foreground
**Is your feature request related to a problem? Please describe.**
Currently, we can use `kyuubi start` to run `kyuubi-server` at background, so we can't manage it's lifecycle by supervision systems like `supervisord`, `systemd`.

**Describe the solution you'd like**
New command `kyuubi run` to run `kyuubi-server` in frontend. It's actually what Trino (formerly PrestoSQL) does. https://trino.io/docs/current/installation/deployment.html#running-trino

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #352](https://github.com/yaooqinn/kyuubi/pull/352)    Feature [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _Why are the changes needed?_

close #312

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

image

### CODE REVIEW

File: bin/kyuubi [hunk 1]
```diff
@@ -20,7 +20,7 @@
 CLASS="org.apache.kyuubi.server.KyuubiServer"
 
 function usage {
-  echo "Usage: ./bin/kyuubi (start|stop|status)"
+  echo "Usage: ./bin/kyuubi (start|run|stop|status)"
```
Comment (yaooqinn) [inline bin/kyuubi:23]: Adding run here make the usage not clear anymore，we need describe each option here

**Review by yaooqinn — CHANGES_REQUESTED** [review]
thanks this change looks  good to me

File: bin/kyuubi (hunk 1 above)
Comment (pan3793) [reply bin/kyuubi]: updated

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.1.0

**Review by yaooqinn — APPROVED** [review]

**Pull request closed without merging**